### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

**Review by amis47 — APPROVED** [review]
AWESOME WORK!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,116 @@
-# eth-client
-A Golang Client Library for Communicating with Ethereum RPC Servers
+eth-client
+==========
+
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg)](https://travis-ci.org/getamis/eth-client)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Go Report Card](https://goreportcard.com/badge/github.com/alanchchen/web3go)](https://goreportcard.com/report/github.com/getamis/eth-client)
+
+A Golang Client Library for Communicating with Ethereum RPC Servers.
```
Comment (tailingchen) [inline README.md]: `A Golang client library to communicate with Ethereum RPC server. `

File: README.md [hunk 2]
```diff
@@ -1,2 +1,116 @@
...
-A Golang Client Library for Communicating with Ethereum RPC Servers
+eth-client
+==========
+
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg)](https://travis-ci.org/getamis/eth-client)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Go Report Card](https://goreportcard.com/badge/github.com/alanchchen/web3go)](https://goreportcard.com/report/github.com/getamis/eth-client)
+
+A Golang Client Library for Communicating with Ethereum RPC Servers. 
+* implements most of JSON-RPC methods plus several client-specific methods
```
Comment (tailingchen) [inline README.md]:
```
Implements most of JSON-RPC methods and several client-specific methods.
Provides a high-level interface to **propose/get validators** on Istanbul blockchain.
Provides a high-level interface to **create private contracts** on Quorum blockchain.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,116 @@
...
+* eth_sendRawTransaction
+* miner_startMining
+* miner_stopMining
+* net_version
+* logs
+* newHeads
+* eth_getLogs
+
+### Istanbul-only JSON-RPC methods
+To use these methods, make sure that you're
```
Comment (tailingchen) [inline README.md]: To use these methods, make sure that
* Server is running on [Istanbul consensus](https://github.com/ethereum/EIPs/issues/650).
* Connect to server through `istanbul.Dial` function (not the original [Geth client](https://github.com/ethereum/go-ethereum/tree/master/ethclient)).

File: README.md [hunk 4]
```diff
@@ -1,2 +1,116 @@
...
+
+
+Methods:
+
+* istanbul_getValidators
+* istanbul_propose
+
+### Quorum-only JSON-RPC methods
+
+To use these methods, make sure that you're
```
Comment (tailingchen) [inline README.md]: To use these methods, make sure that
* Server is running on [Quorum blockchain](https://github.com/jpmorganchase/quorum/wiki)
* Connect to server through `quorum.Dial` function (not the original [Geth client](https://github.com/ethereum/go-ethereum/tree/master/ethclient)).

**Review by tailingchen — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,116 @@
-# eth-client
-A Golang Client Library for Communicating with Ethereum RPC Servers
+eth-client
+==========
+
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg)](https://travis-ci.org/getamis/eth-client)
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
+[![Go Report Card](https://goreportcard.com/badge/github.com/alanchchen/web3go)](https://goreportcard.com/report/github.com/getamis/eth-client)
```
Comment (yutelin) [inline README.md]: https://goreportcard.com/badge/github.com/alanchchen/web3go -> wrong url

**Review by yutelin — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (bailantaotao) [reply README.md]: OK

File: README.md (hunk 4 above)
Comment (bailantaotao) [reply README.md]: OK

File: README.md (hunk 3 above)
Comment (bailantaotao) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (bailantaotao) [reply README.md]: OK

File: README.md (hunk 1 above)
Comment (bailantaotao) [reply README.md]: OK

**Pull request merged**